src/packages/cli/helpers/build.js

```diff
@@ -9,6 +9,24 @@ const { promisify } = require('util')
 const copyFile = promisify(fs.copyFile)
 const lineReplace = require('line-replace')
 
+let prismaClientPlugin = {
```

Comment:
This is an ESBuild plugin that redirects `@prisma/client` imports from Studio to the one we ship with the CLI